supabase/tables/base.ts

```diff
@@ -0,0 +1,12 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Logs } from "ubiquibot-logger";
+
+export class SuperBase {
```

Comment:
```suggestion
export class Supabase {
```

Maybe this is more accurate.